python/dgl/distributed/partition.py

```diff
@@ -242,7 +243,7 @@ def partition_graph(g, graph_name, num_parts, out_path, num_hops=1, part_method=
         node_parts = metis_partition_assignment(g, num_parts)
         client_parts = partition_graph_with_halo(g, node_parts, num_hops, reshuffle=reshuffle)
     elif part_method == 'random':
-        node_parts = dgl.random.choice(num_parts, g.number_of_nodes())
+        node_parts = random_choice(num_parts, g.number_of_nodes())
```

Comment:
what's the between `dgl.random.choice` and `random_choice`. I used many functions in `dgl.random` package.

Reply:
it's the same thing. we cannot use the dgl namespace inside dgl. i didn't write the code correctly.